Week1/homework.js

```diff
@@ -0,0 +1,253 @@
+let mysql = require('mysql');
+
+// establish db
+let db = mysql.createConnection({
+  host: 'localhost',
+  user: 'root',
+  password: 'password',
+  database: 'world',
+});
+
+// connect database
+db.connect(err => {
+  if (err) throw err;
+  console.log('Mysql connected....');
+});
+
+// drop database
+let drop_db = `DROP DATABASE IF EXISTS world`;
+db.query(drop_db, (error, result) => {
+  if (error) throw error;
+  // console.log(result);
+});
+
+// create database
+let create_db = `CREATE DATABASE IF NOT EXISTS world`;
```

Comment:
You can use 'new_world' instead of 'world' here to use the data being loaded by world.sql